katz/src/main/kotlin/katz/Id.kt

```diff
@@ -0,0 +1,31 @@
+package katz
+
+class Id<out A>(val value: A): HK<Id.F, A> {
+
+    class F
```

Comment:
Is it possible to make it so it has 0 instances? how? defining it as an abstract class? any other alternative?